tools/install_ssl_configuration.sh

```diff
@@ -0,0 +1,320 @@
+SCRIPT_PATH="${BASH_SOURCE[0]}"
```

Comment:
you should have a shebang at the beginning: https://stackoverflow.com/a/21613044

```suggestion
#!/usr/bin/env bash

SCRIPT_PATH="${BASH_SOURCE[0]}"
```